Microsoft.Toolkit.Uwp.UI.Controls/Graph/SharePointFiles/SharePointFiles.Properties.cs

```diff
@@ -0,0 +1,245 @@
+        {
+            get { return (int)GetValue(PageSizeProperty); }
+            set { SetValue(PageSizeProperty, value); }
+        }
+
+        internal bool HasMore
+        {
+            get { return (bool)GetValue(HasMoreProperty); }
+            set { SetValue(HasMoreProperty, value); }
+        }
+
+        internal DriveItem SelectedFile
+        {
+            get { return (DriveItem)GetValue(SelectedFileProperty); }
+            set { SetValue(SelectedFileProperty, value); }
+        }
+
+        internal long FileSize
+        {
+            get { return (long)GetValue(SizeProperty); }
+            set { SetValue(SizeProperty, value); }
+        }
+
+        internal string LastModified
+        {
+            get { return (string)GetValue(LastModifiedProperty); }
+            set { SetValue(LastModifiedProperty, value); }
+        }
+
+        private int FileUploading
+        {
+            get
+            {
+                return _fileUploading;
+            }
+
+            set
+            {
+                _fileUploading = value;
+                if (value > 0)
+                {
+                    _status.Text = $"Uploading {value} files...";
+                    _status.TextDecorations = Windows.UI.Text.TextDecorations.None;
+                    _status.Foreground = new SolidColorBrush(Windows.UI.Colors.Black);
+                    if (string.IsNullOrEmpty(_errorMessage))
+                    {
+                        _status.Visibility = Visibility.Visible;
+                    }
+                    else
+                    {
+                        _status.Visibility = Visibility.Collapsed;
+                    }
+
+                    _cancel.Visibility = Visibility.Visible;
+                }
+                else
+                {
+                    _status.Visibility = Visibility.Collapsed;
+                    _cancel.Visibility = Visibility.Collapsed;
+                }
+            }
+        }
+
+        private string ErrorMessage
+        {
+            get
+            {
+                return _errorMessage;
+            }
+
+            set
+            {
+                _errorMessage = value;
+                if (!string.IsNullOrEmpty(value))
+                {
+                    _error.Visibility = Visibility.Visible;
+                    _status.Visibility = Visibility.Collapsed;
+                }
+                else
+                {
+                    if (FileUploading > 0)
+                    {
+                        _status.Visibility = Visibility.Visible;
+                    }
+
+                    _error.Visibility = Visibility.Collapsed;
+                }
+            }
+        }
+
+        private bool IsDetailPaneVisible
+        {
+            get
+            {
+                return (bool)GetValue(IsDetailPaneVisibleProperty);
+            }
+
+            set
+            {
+                SetValue(IsDetailPaneVisibleProperty, value);
+                if (value)
```

Comment:
Bind the boolean property in XAML, there shouldn't methods in this class flipping the properties of controls like this.